android/src/main/java/com/auth0/react/A0Auth0Module.java

```diff
@@ -53,26 +54,23 @@ public String getName() {
     @ReactMethod
     public void showUrl(String url, boolean closeOnLoad, Callback callback) {
         final Activity activity = getCurrentActivity();
+        final Uri parsedUrl = Uri.parse(url);
         this.callback = callback;
 
         try {
-            if (activity != null) {
-                CustomTabsIntent.Builder builder = new CustomTabsIntent.Builder();
-                CustomTabsIntent customTabsIntent = builder.build();
-                customTabsIntent.launchUrl(activity, Uri.parse(url));
-            } else {
-                final Intent intent = new Intent(Intent.ACTION_VIEW);
-                intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
-                intent.setData(Uri.parse(url));
-                getReactApplicationContext().startActivity(intent);
+            if (activity != null) AuthenticationActivity.authenticateUsingBrowser(activity, parsedUrl);
```

Comment:
let's be consistent and add the braces `{ }` here, please

Reply:
Added in https://github.com/auth0/react-native-auth0/pull/350/commits/b5e827bce2820ffb8930272c044f1ccf12e5091e